Adding custom resource error message when deployed from studio (#5924)
aws-amplify/docs: src/pages/console/adminui/extend-cli.mdx

</Callout>

### Overrides and Custom Resources

```console
A flag to disable overrides has been detected, please deploy from a different environment. at buildOverrideDir
```
```console
A flag to disable custom resources has been detected, please deploy from a different environment. at buildResource
```

> The above errors are encountered if an Amplify project has an `override.ts` file or [custom resources](https://docs.amplify.aws/cli/custom/cdk/) deployed in the project. Amplify Studio does not support deploying projects containing override and custom resources.

Reviewer: ```suggestion
The errors mentioned above occur during deployment if your Amplify project contains an `override.ts` file or [custom resources](https://docs.amplify.aws/cli/custom/cdk/) deployed. Amplify Studio does not support deploying projects with override files or custom resources.
```

> The errors mentioned above occur during deployment if your Amplify project contains an `override.ts` file or [custom resources](https://docs.amplify.aws/cli/custom/cdk/) deployed. Amplify Studio does not support deploying projects with override files or custom resources.

Reviewer 2: ```suggestion
The errors mentioned above occur during deployment if your Amplify project contains an `override.ts` file or [custom resources](https://docs.amplify.aws/cli/custom/cdk/) are deployed. Amplify Studio does not support deploying projects with override files or custom resources.
```

> To resolve this error, remove the `override.ts` file or custom resources and run `amplify push`. Alternatively, you can use Amplify CLI to build and push your resources. But you will not be able to deploy with Amplify Studio.

Reviewer: ```suggestion
To resolve this error, remove the `override.ts` file or any custom resources and run `amplify push`. Alternatively, you can use Amplify CLI to build and push your resources; however, deploying with Amplify Studio will not be possible.
```

> To resolve this error, remove the `override.ts` file or any custom resources and run `amplify push`. Alternatively, you can use Amplify CLI to build and push your resources; however, deploying with Amplify Studio will not be possible.

Reviewer 2: ```suggestion
To resolve this error, remove the `override.ts` file and any custom resources from your project, then run `amplify push`. Alternatively, you can use Amplify CLI to build and push your resources; however, deploying with Amplify Studio will not be possible.
```
